src/python/pants/invalidation/cache_manager.py

```diff
@@ -172,20 +172,34 @@ def __init__(self, cache_manager, target, cache_key):
     super(VersionedTarget, self).__init__(cache_manager, [self])
     self.id = target.id
 
+  def _results_dir_prefix(self, root_dir):
```

Comment:
Oh - one other motivation - it included the change you just added, so we at minimum agreed on that part  : )